### PR TITLE
fix npm index to display package.json on package folder

### PR DESCRIPTION
**Pull request by shokakucarrier**
We can change "package.json" in template easily if we decide to use another json file to present the content

### CODE REVIEW

File: charon/utils/strings.py [hunk 1]
```diff
@@ -2,3 +2,8 @@ def remove_prefix(string: str, prefix: str) -> str:
     if string and prefix and string.startswith(prefix):
         return string[len(prefix):]
     return string
+
+def remove_sufix(string: str, sufix: str) -> str:
```
Comment (ligangty) [inline charon/utils/strings.py:6]: Um... where is this remove_suffix func used? I don't see it's called anywhere.

**Review by ligangty — COMMENTED** [review]
LGTM except 2 comments.

File: charon/utils/strings.py (hunk 1 above)
Comment (shokakucarrier) [reply charon/utils/strings.py:6]: I’ll remove this

File: charon/constants.py [hunk 2]
```diff
@@ -143,3 +145,35 @@
...
+  <header>
+    <h1>{{ index.header }}</h1>
+  </header>
+  <hr/>
+  <main>
+    <ul style="list-style: none outside;" id="contents">
+    {% for item in index.items %}{% if item.endswith("/") %}{% if item.startswith("@") or item.startswith("..") %}
+        <li><a href="{{ item }}index.html" title="{{ item }}">{{ item }}</a></li>
+    {% else %}
+        <li><a href="{{ item }}package.json" title="{{ item }}">{{ item }}</a></li>
```
[the remark this answers is not in the document]
Comment (shokakucarrier) [reply charon/constants.py:170]: so everything except folder/index.html will be redirected to package.json?

**Comment by ligangty** [comment]
Seems unittests failed.

File: tests/test_npm_index.py [hunk 3]
```diff
@@ -25,6 +25,21 @@
 import boto3
 import os
 
+TEST_BUdCKET = "npm_bucket"
```
Comment (jdcasey) [inline tests/test_npm_index.py:28]: minor typo...

**Review by jdcasey — APPROVED** [review]
LGTM. I'm not sure I understand the code deeply enough to be definitive, but it seems to make sense.

**Pull request merged**